gulp-tasks/utils/npmRunScript.js

```diff
@@ -0,0 +1,13 @@
+const path = require('path');
+const spawn = require('./spawn-promise');
+
+const npmRunScript = (scriptName) => {
```

Comment:
If `lint` and `genGQLTypes` get inlined, this could also get folded into `build`, or at least `spawn-promise` gets inlined here.

Reply:
Generally, there are useful enough that I reckon we'll end up wanting to re-use these.